tests/components/webostv/test_config_flow.py

```diff
@@ -337,7 +330,7 @@ async def test_reauth_errors(
     assert result["type"] is FlowResultType.FORM
     assert result["step_id"] == "reauth_confirm"
 
-    monkeypatch.setattr(client, "connect", Mock(side_effect=side_effect))
+    client.connect = AsyncMock(side_effect=side_effect())
```

Comment:
```suggestion
    client.connect.side_effect = side_effect
```
I believe this works as well